src/openjd/sessions/_runner_base.py

```diff
@@ -21,8 +21,7 @@
 from openjd.model.v2023_09 import Action as Action_2023_09
 from ._embedded_files import EmbeddedFiles, EmbeddedFilesScope, write_file_for_user
 from ._logging import log_subsection_banner
-from ._os_checker import is_posix, is_windows
-from ._powershell_generator import generate_exit_code_wrapper
+from ._os_checker import is_posix  # , is_windows
```

Comment:
nit: commented out code
```suggestion
from ._os_checker import is_posix
```